language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/Filter.java

```diff
@@ -18,22 +18,33 @@ public static Filter fromProto(TransactionFilterOuterClass.Filters filters) {
   public abstract TransactionFilterOuterClass.Filters toProto();
 
   /**
-   * Settings for including an interface in {@link InclusiveFilter}. There are two values: {@link
-   * #INCLUDE_VIEW} and {@link #HIDE_VIEW}.
+   * Settings for including an interface in {@link InclusiveFilter}. There are four possible values:
+   * {@link #HIDE_VIEW_HIDE_CREATED_EVENT_BLOB} and {@link #INCLUDE_VIEW_HIDE_CREATED_EVENT_BLOB}
+   * and {@link #HIDE_VIEW_INCLUDE_CREATED_EVENT_BLOB} and {@link
+   * #INCLUDE_VIEW_INCLUDE_CREATED_EVENT_BLOB}.
    */
-  public static final class Interface {
-    public final boolean includeInterfaceView;
-    // add equals and hashCode if adding more fields
+  public enum Interface {
```

Comment:
These types should probably be `static`.

```suggestion
  public static enum Interface {
```

Reply:
IJ was suggesting that is not necessary for inner enums. But fine by me to apply